modules/cli-manager-updating-plugin.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-updating-plugin_{context}"]
+= Updating CLI plugins with the {cli-manager}
+
+You can update a plugin that was installed for the OpenShift CLI (`oc`) with the {cli-manager}.
```

Comment:
Maybe specify that you're updating them to a newer version?

Question (maybe for @ardaguclu too) - does the admin have to do something to make these newer versions available? I don't see anything on this in the doc set yet.

Reply:
Yes, cluster admin should apply the Plugin containing the new version and corresponding new binary to make it available by users.